packages/sdk/src/utils/typedData.ts

```diff
@@ -33,7 +33,7 @@ export function getUpdateDepositTypedData({
     },
     primaryType: "UpdateDepositDetails",
     message: {
-      depositId,
+      depositId: Number(depositId), // TODO: support new UpdateDepositDetails using bytes32 and uint256 for recipient & depositId, respectively
```

Comment:
Should we add a `getUpdateDepositTypedDataV3_5` or similar that uses the new types? And mark this one as deprecated

Reply:
Yes I don't think we can support both unless the sdk knows which version the contracts are on. So maybe deprecate the old one and only support new